src/css/pages/appliances.css

```diff
@@ -21,13 +21,23 @@ fieldset {
   margin: 2em 0em;
   padding: 1em;
   border: 0;
-  background-color: #B2C9D6;gf
+  background-color: #B2C9D6;
 }
 
 main {
   margin: 1em;
 }
 
+main a {
+  color: #5C637A;
+  opacity: .65;
+  transition: .3s ease-in-out;
```

Comment:
This transitions all properties of the anchor. Could cause problems when other properties get modified in states